testing/it/ui-js/src/content/jcr_root/apps/core/wcm/tests/test-cases/Title/v1/Title.js

```diff
@@ -267,4 +267,33 @@ window.CQ.CoreComponentsIT.Title.v1 = window.CQ.CoreComponentsIT.Title.v1 || {}
         ;
     };
 
+    /**
+     * Test: set link on title
+     */
+    title.tcSetLink = function(tcExecuteBeforeTest, tcExecuteAfterTest) {
+        return new h.TestCase("Set Link", {
+            execBefore: tcExecuteBeforeTest,
+            execAfter: tcExecuteAfterTest })
+
+            // open the config dialog
+            .execTestCase(c.tcOpenConfigureDialog("cmpPath"))
+            // enter the link
+            .simulate("foundation-autocomplete[name='./linkURL'] input[type!='hidden']", "key-sequence",
+                { sequence: c.rootPage + "{enter}" })
+            // save the dialog
+            .execTestCase(c.tcSaveConfigureDialog)
+
+            // switch to content frame
+            .config.changeContext(c.getContentFrame)
+            // click on the title
+            .click("div.cmp-title a", { expectNav: true })
```

Comment:
and here, let's use the BEM element, `.cmp-title__link`

Reply:
Done